src/main/java/frc/trigon/robot/constants/RobotConstants.java

```diff
@@ -8,12 +8,13 @@ public class RobotConstants {
     private static final boolean
             IS_SIMULATION = true,
             IS_REPLAY = false;
+    private static final RobotHardwareStats.ReplayType REPLAY_TYPE = IS_SIMULATION ? RobotHardwareStats.ReplayType.SIMULATION_REPLAY : RobotHardwareStats.ReplayType.REAL_REPLAY;
```

Comment:
I don't think this is what Yishai meant. The new logic is meant to make things simpler, this makes it more complicated. Just have a car for isReal and a var for replayType